package.json

```diff
@@ -46,6 +46,7 @@
     "test-watch": "karma start --auto-watch --no-single-run"
   },
   "dependencies": {
+    "connect-modrewrite": "^0.9.0",
```

Comment:
This wasn't your change....it shouldn't show up in the PR diff. I'm convinced there is something messed up with your Git or the way you're pulling/merging code that is causing a problem. If you look at the "Commits" in the PR, it shows way more commits than you actually did on your branch....